azuread/resource_application_test.go

```diff
@@ -642,18 +638,19 @@ func testAccADApplication_basicEmpty(ri int) string {
 	return fmt.Sprintf(`
 resource "azuread_application" "test" {
   name                    = "acctest-APP-%[1]d"
+  homepage                = "https://homepage-%[1]d" // Cannot be unset once set, see https://github.com/Azure/azure-sdk-for-go/issues/10463
```

Comment:
Should it be force new then?

Reply:
ForceNew would make sense but there is a case where you can go from no homepage to some homepage, e.g. from
```hcl
resource "azuread_application" "test" {
  name = "test"
}
```
to
```hcl
resource "azuread_application" "test" {
  name = "test"
  homepage = "https://test"
}
```
which this works, and I think we should enable that.

Reply:
we could use customize diff to conditionally force new?